app/code/Magento/NewRelicReporting/Model/NewRelicWrapper.php

```diff
@@ -41,6 +41,19 @@ public function reportError($exception)
         }
     }
 
+    /**
+     * Wrapper for 'newrelic_set_appname'
+     *
+     * @param string $appName
+     * @return void
+     */
+    public function setAppName($appName)
```

Comment:
I'd like to define variable type to "string" there, we're not supporting php 5.6 anymore